.github/workflows/command-line-docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate command line documentation
+
+on:
+  push:
+    branches:
+      - main
+  merge_group:
+    types: [checks_requested]
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number && format('pr{0}', github.event.number) || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  CheckSymbols:
+    runs-on: ubuntu-24.04
+
+    timeout-minutes: 10
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo --preserve-env apt-add-repository --yes ppa:mir-team/dev
+          sudo apt-get install --yes libclang1-18 python3-clang-18
+          echo "MIR_SYMBOLS_MAP_GENERATOR_CLANG_SO_PATH=/usr/lib/llvm-18/lib/libclang-18.so.1" >> $GITHUB_ENV
+          echo "MIR_SYMBOLS_MAP_GENERATOR_CLANG_LIBRARY_PATH=/usr/lib/llvm-18/lib" >> $GITHUB_ENV
```

Comment:
Since these are constant, might it be simpler to use the [`env:` stanza at the job level](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idenv)?

Reply:
Removed